nbs/models.tft.ipynb

```diff
@@ -15,20 +15,14 @@
    "source": [
```

Comment:
<p>This equation is not compiling: \begin{align} \mathbf{E}^{(h)}<em>{t} &amp;= [e^{(h)}</em>{t,1},\dots,e^{(h)}<em>{t,j},\dots,e^{(h)}</em>{t,n_{h}}] \ \mathbf{\tilde{e}}^{(h)}<em>{t,j} &amp;= \mathrm{GRN}(e^{(h)}</em>{t,j}) \end{align}</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/293/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
I will pass them to main, to see if the documentation compiles the latex correctly.